lib/jsanalyze.js

```diff
@@ -171,7 +173,34 @@ exports.analyzeJs = function analyzeJs(contents, opts) {
 	// transpile
 	if (opts.transpile) {
 		options.plugins.push(require.resolve('./global-this'));
-		options.presets.push([ env, { targets: opts.targets } ]);
+		options.presets.push([ env, { targets: opts.targets, useBuiltIns: 'entry' } ]);
+
+		// install polyfill
+		let child = null;
+		if (opts.resourcesDir) {
+			if (process.platform === 'win32') {
```

Comment:
Seeing as we have the dependency added to the package.json, could we potentially copy it over to the app, it might have fewer potential problems that spawning npm

Reply:
Second, I think this is a case where you'd want to use `child_process.fork()` since we really want to fork off another node process that is the same node install as currently being used. Obviously, that's an async API, so there's an issue there...

Reply:
Also, we discussed on Teams chat - but it may be better to just copy the babel-polyfill dependency tree over instead of doing an npm install. This could possibly achieve that: https://www.npmjs.com/package/copy-node-modules